core_weekly/template.py

```diff
@@ -315,7 +315,7 @@ def build_merged_pull_requests(self, result, category_blacklist):
             for category, items in sorted_category_items.items():
                 category_name = CATEGORIES[category] if category in CATEGORIES.keys() else category
 
-                if (category in category_blacklist):
+                if category in category_blacklist:
```

Comment:
```suggestion
                if category in CATEGORIES_REJECT_LIST:
```

Reply:
It's a const, it doesn't need to be injected in the template. 
Do like CATEGORIES, it's enough

Reply:
Would'nt it be better to keep `build_merged_pull_requests()` configurable (so we can pass whatever denylist we want to it) ?

If for example I want to unit-test `build_merged_pull_requests()` ; by using a parameter I can check different scenarii

Reply:
This is something you're already able to do with Python, it helps you to patch constant:
https://docs.python.org/3/library/unittest.mock.html#where-to-patch

Reply:
Thank you for the insights 😄 always great to learn

I will apply the feedback